src/modules/toast/sagas.ts

```diff
@@ -0,0 +1,41 @@
+import { all, takeEvery, put } from 'redux-saga/effects'
+import { Network } from '@dcl/schemas'
+import { toastSaga as baseToastSaga } from 'decentraland-dapps/dist/modules/toast/sagas'
+import { showToast } from 'decentraland-dapps/dist/modules/toast/actions'
+import { getMetaTransactionFailureToast } from './toasts'
+import {
+  InitiateWithdrawalFailureAction,
+  INITIATE_WITHDRAWAL_FAILURE,
+  TransferManaFailureAction,
+  TRANSFER_MANA_FAILURE,
+} from '../mana/actions'
+
+export function* toastSaga() {
+  yield all([baseToastSaga(), customToastSaga()])
+}
+
+function* customToastSaga() {
+  yield takeEvery(TRANSFER_MANA_FAILURE, handleTransferManaFailure)
```

Comment:
Should we need to add `DEPOSIT_MANA_FAILURE`, the actions related to transfer mana on polygon, and the init of the withdrawal process?

Reply:
Transferring MANA on polygon is handled by this handler, and initiating the withdrawal is handled in the line below. I didn't add the deposit because that one is a regular tx on Ethereum, afaik the promise would only fail if the user rejects it (and we ignore those) or if an address/network is wrong (shouldn't happen if everything is configured right).